hack/multi-node/user-data.sample

```diff
@@ -19,36 +19,58 @@ coreos:
           [Unit]
           Requires=flanneld.service
           After=flanneld.service
+    - name: kubelet-initial-config.service
+      enable: true
+      command: start
+      content: |
+        [Service]
+        ExecStart=/home/core/install-initial-kubelet-config
```

Comment:
:(

So my goal is that the user-data we need to provide is only `kubelet.service` and a `kubeconfig`. So with that in mind, what about just setting two Environment variables, and an optional (`=-`) EnvironmentFile which the node-agent can use. For example:

```
Environment=KUBELET_ACI=quay.io/coreos/hyperkube
Environment=KUBELET_VERSION=v1.3.6_coreos.0
EnvironmentFile=-/etc/kubernetes/kubelet.env
```

We can revisit this later if it's really not the behavior we want - but I'd like to start clean/simple and work from there.